### PR TITLE
Forward max width and height args to rpack in pack_image_nest

### PR DESCRIPTION
**Pull request by QuantuMope**
Simple PR that allows `max_width` and `max_height` args to be forwarded to `rpack.pack`. Needed to control the dimensions of the packed image in cases where multiple minimal area solutions exist.

Also, fixed some minor spelling errors concerning `interploation --> interpolation`.

### CODE REVIEW

File: alf/summary/render.py [hunk 1]
```diff
@@ -175,7 +181,8 @@ def pack_image_nest(cls, imgs):
         # first get all images' sizes (w,h)
         sizes = [(i.shape[1], i.shape[0]) for i in imgs]
         # call rpack for an approximate solution: [(x,y),...] positions
-        positions = rpack.pack(sizes)
+        positions = rpack.pack(
+            sizes, max_width=max_width, max_height=max_height)
```
Comment (hnyu) [inline alf/summary/render.py]: What if there is no such a solution given the max_width and max_height? Will that crash the function?

Comment (QuantuMope) [reply alf/summary/render.py]: In that case, a `rpack.PackingImpossibleError` will be thrown. Do you prefer having the user handle the error or catching this and rerunning without limits perhaps?

Comment (hnyu) [reply alf/summary/render.py]: > In that case, a `rpack.PackingImpossibleError` will be thrown. Do you prefer having the user handle the error or catching this and rerunning without limits perhaps?

I think the second option is better.

Comment (QuantuMope) [reply alf/summary/render.py]: Changed the behavior to catch the error and rerun with constraints dropped. A warning is also printed when this occurs. PTAL.

**Review by hnyu — APPROVED** [review]

**Review by breakds — APPROVED** [review]

**Pull request merged**